### PR TITLE
Math magicians 2nd milestone 

### PR DESCRIPTION
**Pull request by marcosmerida**
- React set up.
- React components.
- webpack.
- CSS.

### CODE REVIEW

File: src/component/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import './calculator.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+function Calculator() {
```
Comment (eliasecasta) [inline src/component/Calculator.js]: - [x] The `Calculator.js` component should be class-based, please refactor it to comply with the project requirements.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,8 @@
-import logo from './logo.svg';
+import Calculator from './component/Calculator';
 import './App.css';
 
 function App() {
```
Comment (eliasecasta) [inline src/App.js]: - [x] The `App.js` component should be class-based, please refactor it to comply with the project requirements.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @marcosmerida, I am Elias, your reviewer for this PR

Well done so far with your project!

Your calculator design looks amazing! I would only suggest you make it bigger.

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] Please use the Microverse template readme given to you for this milestone, currently, you are using the one provided by React but it is needed for this milestone to have all the project information necessary.
- [x] Please remove all the boilerplate files (react logos, favicon and the index.css file) in order to comply with the project requirements.
- [x] Please check below for extra requirements in this PR.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] Please make your calculator bigger to make it look better and easier to use.

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Davidosky007** [comment]
## Hi @marcosmerida 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
* As the previous reviewer stated that all boilerplate files be removed your folder still has some boilerplate file 
image

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @marcosmerida, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**